cluster/cluster.pb.go

```diff
@@ -1,8 +1,12 @@
+// Define the syntax version for this protobuf file, which is proto3.
+
 // Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
See this comment.

Reply:
Done. Removed from the proto file therefore, automatically removed here.